tests/examples/pacer/nef/s3/paeb_3.json

```diff
@@ -4,45 +4,22 @@
   "court_id": "paeb",
   "dockets": [
     {
-      "case_name": "Tri-State Paper, Inc. v. Foodarama Caterers Inc.",
```

Comment:
Also this seems like a different email example. Could we add  it a separate one?